packages/@glimmer/destroyable/index.ts

```diff
@@ -159,11 +161,17 @@ export function destroy(destroyable: Destroyable) {
   meta.state = DESTROYING_STATE;
 
   iterate(children, destroy);
-  iterate(eagerDestructors, (destructor) => destructor(destroyable));
-  iterate(destructors, (destructor) => scheduleDestroy(destroyable, destructor));
+  iterate(eagerDestructors, (destructor) => {
```

Comment:
why add bodies to these functions?

Reply:
I could have used the `void` prefix, but TL;DR if the expression returns non-void, the stricter lint settings don't let you implicitly return the value via an arrow.